Modules/itertoolsmodule.c

```diff
@@ -3386,7 +3387,7 @@ count_nextlong(countobject *lz)
 
     long_cnt = lz->long_cnt;
     if (long_cnt == NULL) {
-        /* Switch to slow_mode */
+        /* Switching from fast mode */
```

Comment:
Why change the comment?

Reply:
This function, `count_nextlong`, is slow mode only. We get here if we already decided to use slow mode (in `next_count`), so the comment was a bit confusing. What's happening here is that this `if` checks whether this is the first time we switched from fast mode (so need to initialize the "slow counter"). IMHO this is more correct.

Reply:
Reverted